src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayForm.tsx

```diff
@@ -59,16 +78,100 @@ export const V2V3PayForm = ({
   const canAddMoreStickers =
     (stickerUrls ?? []).length < ProjectPreferences.MAX_IMAGES_PAYMENT_MEMO
 
-  useEffect(
```

Comment:
by setting the form.beneficiary to userAddress by default, we can also remove this bit

Reply:
clean affffff